ios/Handlers/RNPanHandler.m

```diff
@@ -206,6 +206,16 @@ - (void)configure:(NSDictionary *)config
   APPLY_FLOAT_PROP(failOffsetYStart);
   APPLY_FLOAT_PROP(failOffsetYEnd);
   
+  if (@available(iOS 13.4, *)) {
+    id prop = config[@"iosEnableTouchpadTwoFingerGesture"];
+    if(prop != nil){
+      bool iosEnableTouchpadTwoFingerGesture = [RCTConvert BOOL:prop];
```

Comment:
I think we can convert in the same line with `config[(...)]` if `[RCTConvert BOOL:nil]` returns `nil`.

Reply:
Updated and works